app/grandchallenge/products/templates/products/product_detail.html

```diff
@@ -192,16 +192,16 @@ <h1 class="mb-1">{{product.product_name}}</h1>
             </tr>
           </thead>
           <tr>
-            <td class="text-muted">Peer reviewed papers regarding performance as commercially available</td>
-            <td>{{ product.tech_peer_papers|linebreaksbr }}</td>
+            <td class="text-muted">Peer reviewed papers on performance</td>
+            <td>{{ product.tech_peer_papers|safe|linebreaksbr }}</td>
```

Comment:
Using `safe` is a security risk. Is it really needed? If so, use the `clean` filter instead, this will clean any dangerous html. At the top of the html file add `{% load bleach %}` and then replace `safe` with `clean`.

Reply:
fixed, clean indeed also works